### DISCUSSION TITLE
Azuracast Forum

### DISCUSSION THREAD

**Discussion by Mixnetwork** [AzuraCast/AzuraCast#4293] [General]
Hi all,
I noticed though many people use Azuracast there exist no Azuracast Forum.
I installed a Forum specially fpr Azuracast, because I think it would be a good idea.
I want to  use 5 basic languages eg: Englisch, German,Dutch, French and Italian.
Now i can not do this by myself, so i need some help.

If there are any members here who want to help to build up this forrum, i would be glad to here from you. Experience with SMF Forums or also some other Forum Software woud be nice.

you can contact me over: facebook : https://www.facebook.com/raymond.ummels
Or write me an e-mail; info@fleamingwelle.de or just over the basic forum which is already installed.
the url of the forum is https://azuracast-forums.com

Notice: I just started to create the basic Forum, no topics are yet posted, first we need to make the basics in each Language.
I can do German , Dutch and Englisch but this would be too much for me, i do not have the time to do all 3 languages

**Comment by BusterNeece** [comment]
@Mixnetwork While I appreciate the effort involved in setting up a forum for AzuraCast, I'm not sure I like the way this was done.

For one, our team wasn't informed or consulted about this until after you already not only bought the `azuracast-forums.com` domain name, but also installed a forum there and had set it up with a title and threads implying it's already the "official" forum of our project.

Second, it appears your Terms of Service were borrowed from a religious organization's closed forum, and thus includes language about requiring participation in the Catholic church with the name of the church somewhat find-and-replaced to say "AzuraCast", implying that our membership terms have religious requirements, which they very clearly do not.

Given the advent of GitHub Discussions and the existence of GitHub Issues, our Reddit community and our Discord community, we don't feel there is any need for an additional forum to exist for our community members to communicate. Our small team size prevents us from having the human resources necessary to properly moderate any such forum.

Even if we did desire to set up such a forum, we would do so on our own managed hosting infrastructure with our own knowledge of modern PHP best practices, likely meaning we wouldn't be using a Simple Machines Forum in our own implementation.

In other words, there is no way that we could reasonably take the forums you've created here and make them our "official" forum. We greatly wish you had asked us about this before going and purchasing a domain name and setting up the forum, as we could've told you we weren't interested in having something like this and saved you a fair amount of trouble.

As it stands right now, while we aren't necessarily in a position to tell you to take down the forums you've created, I can say we aren't interested in using them as our official forums and wouldn't reference them in any of our official documentation, so your best bet may be to just spin down the server hosting them and repurpose those resources elsewhere.

I feel bad for having to come down rather hard on this particular project, but we absolutely should have been consulted on this earlier than this stage in the project's existence.
